todo-app/src/App.js

```diff
@@ -1,13 +1,109 @@
 import React, { Component } from 'react';
+import Task from './Task';
+//import tasks from './task.json';
+import TaskForm from './taskForm';
+import Moment from 'moment';
 
-class App extends Component {
+export default class App extends Component {
+
+  constructor(props) {
+    super(props);
+	let tasks = [];
+	if (localStorage.tasks != null) { 
```

Comment:
Careful here. You should try to use strict equality checks wherever possible. In fact, `localStorage.tasks` will never be `null` unless you set it to `null`, it will be `undefined` by default. The only reason this loose comparison works is because `undefined` and `null` are both falsey.

You can check out the [MDN docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) for some good ideas about how to check against `undefined` in JavaScript.